bool detected
BoundingBox boundingbox
Keypoint[2] keypoint